TestCase/Managers/GameManager.cs

```diff
@@ -0,0 +1,100 @@
+﻿namespace TestCase.Managers
+{
+    public class GameManager
+    {
+        private readonly BoardManager _boardManager;
+        private readonly InputManager _inputManager;
+
+        private bool isPlayerXTurn = true;
+        private bool isPlayerOTurn = false;
```

Comment:
Could we find a way to only have 1 variable keeping track of whose turn it is. We only need to check if the player is 1 player if not it will defo be the other player.